projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_base.py

```diff
@@ -216,7 +216,7 @@ def test_ingest_payload_and_post_ingestion_operation():
     )
     ingester_base._ingester.post_ingest_process.assert_called_with(
         payload=test_aggregated_payload,
-        ingestion_result=test_ingestion_result,
+        ingestion_metadata=test_ingestion_result,
         exceptions=[],
```

Comment:
look to cleanup any leftover params from previous pre- or post- ingestion format

Reply:
I believe I have cleaned up the leftovers.